### DISCUSSION TITLE
Proxy support for SSH

### DISCUSSION THREAD

**Discussion by huo-feng-ding** [1Remote/1Remote#430] [Ideas]
像SecureCRT这样加一个firewall，通过这个再进行连接。不然现在这个软件用不了，无法连接上服务器。

![QQ图片20210415183609](https://user-images.githubusercontent.com/3919535/114856415-aab08c00-9e19-11eb-8c90-b4f1891101e8.png)

**Comment by VShawn** [comment]
请问这个 firewall 是什么东西？有什么作用，我没用过 SecureCRT  不太清楚这个设置

**Comment by huo-feng-ding** [comment]
是一个代理，里边的配置有如下

![QQ图片20210416094244](https://user-images.githubusercontent.com/3919535/114959696-1d604c80-9e98-11eb-9c2a-f0ad2b3d59cf.png)

**Comment by VShawn** [comment]
原来如此，代理功能在规划考虑中，暂时没搞清楚应该如何进行开发。

https://github.com/VShawn/PRemoteM/issues/105
